### PR TITLE
update README

### PR DESCRIPTION
**Pull request by Fazziekey**
- update README
- fix environment.yml, now can install the torch version with cuda

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,2 +1,6 @@
+--find-links https://download.pytorch.org/whl/cu113/torch_stable.html
+torch==1.12.1+cu113 
+torchaudio==0.12.1+cu113
+torchvision==0.13.1
 einops
-colossalai
\ No newline at end of file
+colossalai
```
Comment (feifeibear) [inline requirements.txt:6]: add a version number
colossalai==0.1.8

File: README.md [hunk 2]
```diff
@@ -34,8 +38,6 @@ bash scripts/patch_openmm.sh
 You can get the FastFold source and install it with setuptools:
```
Comment (Gy-Lu) [inline README.md:38]: Maybe till this line, I have got the source code?

File: requirements.txt [hunk 3]
```diff
@@ -1,2 +1,6 @@
+--find-links https://download.pytorch.org/whl/cu113/torch_stable.html
```
Comment (Gy-Lu) [inline requirements.txt:1]: If this is a way of installing fastfold to the current environment, remind it in readme.md

File: README.md (hunk 2 above)
Comment (Fazziekey) [reply README.md:38]: fixed

File: requirements.txt (hunk 3 above)
Comment (Fazziekey) [reply requirements.txt:1]: fixed

**Pull request merged**